src/commands/distribute/releases/add-destination.ts

```diff
@@ -0,0 +1,135 @@
+import { AppCommand, CommandResult, ErrorCodes, failure, help, success, shortName, longName, required, hasArg, defaultValue } from "../../../util/commandline";
+import { AppCenterClient, models, clientRequest, ClientResponse } from "../../../util/apis";
+import { out } from "../../../util/interaction";
+import { inspect } from "util";
+import * as _ from "lodash";
+import { DefaultApp } from "../../../util/profile";
+
+const debug = require("debug")("appcenter-cli:commands:distribute:releases:add-destination");
+
+const ValidDestinationTypes = ["group", "tester"];
+
+class AddDestinationError extends Error {
+  constructor(message: string, public errorCode: ErrorCodes) {
+    super(message);
+  }
+}
+
+@help("Distributes an existing release to an additional destination")
+export default class AddDestinationCommand extends AppCommand {
+
+  @help("Release ID")
+  @shortName("r")
+  @longName("release-id")
+  @required
+  @hasArg
+  public releaseId: string;
+
+  @help("Destination Type: group or tester")
```

Comment:
Is it worth setting the help text to display the valid values from `ValidDestinationTypes` in case this changes in future?